sympy/vector/coordsys.py

```diff
@@ -54,8 +55,10 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             Used for simple str printing.
 
         """
-        from sympy.abc import a, x, y, z, u, v, w
-
+        a = Dummy('a')
```

Comment:
It is also possible to define several dummies on the same line: `c, d = symbols('c,d', cls=Dummy)`.